### PR TITLE
fix(challenges): add class to backend challenge wrapper, fix basic no…

### LINKED ISSUE #13163: [beta] blockquote "code example" formatting in backend challenges
#### Challenge Name

All back end challenges.

Since the backend challenges are a new addition with the expansion, they are missing some things that normal challenges have from a styling perspective. For example, the current challenges have code examples wrapped in `<code>` tags like so:
image
However, what would look better, and be more consistent with our other challenges is for them to be wrapped in `<blockquote>`s. But these challenges are not wrapped in a container with the `.challenge-instructions` class, so if we were to add a blockquote, it would not appear with the styles we added to them for other challenges. To fix this, I suggest that we:

- Add the class `.challenge-instructions` to the container surrounding the challenge description in the correct component that renders these

### LINKED ISSUE #13168: [beta] confusing instructions in Root-level Request Logger challenge - Basic Node
#### Challenge Name

http://beta.freecodecamp.com/en/challenges/basic-node-and-express/implement-a-rootlevel-request-logger-middleware

#### Issue Description

The instructions currently say:

> Build a simple logger. For every request, it should log in the console a string taking the following format : method path - ip.  

But since there is no example provided it is not immediately clear the exact solution the tests will be looking for. Even with an example, I think it could be a bit confusing unless we explicitly spell it out. Even though I had the basics of the code down, I had to look at the solution to get the specific formatting correct. 

I was able to get that there should be a dash in between "path" and "ip", but one might think the rest of the spacing is there just to make things readable, not because it is necessarily required. 

A better solution might be to change the instructions to say:

> Build a simple logger. For every request, it should log in the console a string taking the following format: `method path - ip`. An example would look like: `GET /json - ::ffff:127.0.0.1`. Note that there is a space between `method` and `path` and that the dash separating `path` and `ip` is surrounded by a space on either side...

**Note:** There are some other issues with this challenge as well from a grammar point of view, so could probably all be fixed with one PR

### PR DESCRIPTION
**Pull request by no-stack-dub-sack**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #13163
Also closes #13168

#### Description

- added the class `.challenge-instructions` to the `<Row>` component surrounding the function that renders challenge instructions in the new backend challenge component
- changed all the code examples in Basic Node and Express section to `<blockquote>`s to exemplify the changes and for consistency with rest of challenge seed
- this will also allow us to use `<dfn>` tags in this section which previously would not have formatted correctly

### NOTE: 
I think this may cause some merge conflicts in other PRs if merged before them. #13155 and #13162 should be merged first to avoid this I believe.

### CODE REVIEW

File: seed/challenges/05-apis-and-microservices/basic-node-and-express.json
Comment (JosephLivengood) [inline seed/challenges/05-apis-and-microservices/basic-node-and-express.json]: I recommend using <pre> with \n. This way you can preserve indentation and it's formatted as code. See attached for what it looks like:

screen shot 2017-02-05 at 11 55 03 pm

Comment (no-stack-dub-sack) [inline seed/challenges/05-apis-and-microservices/basic-node-and-express.json]: @JosephLivengood Hey Joseph, actually, wrapping it in a blockquote and adding the class that I've added to the challenge description wrapper is what will give it the appearance of how code examples look in other challenges. Even though they appear to be wrapped in pre at first glance, stylized blockquotes are the FCC standard. See the code [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/client/less/challenge.less#L75) to check how the effect is achieved.

**Comment by raisedadead** [comment]
@no-stack-dub-sack blocking for a merge post the others?

**Comment by no-stack-dub-sack** [comment]
@raisedadead Yeah, sorry, meant to add the label. Once the 2 referenced above are merged, we can remove this label and review.

**Comment by raisedadead** [comment]
@no-stack-dub-sack I guess we are good to go after addressing the review comments now?

**Comment by no-stack-dub-sack** [comment]
@raisedadead Yeah, if anyone would like to review at this point, the two holding it up have been merged

File: seed/challenges/05-apis-and-microservices/basic-node-and-express.json
Comment (raisedadead) [inline seed/challenges/05-apis-and-microservices/basic-node-and-express.json]: `app.METHOD(PATH, HANDLER)`

Can we wrap these and similar things in `<code></code>` for better UX and consistency.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by raisedadead** [comment]
@no-stack-dub-sack these changes LGTM, except for some formatting overhaul. Would you be interested in doing so via this PR.

Or we could take up later, I'll leave the choice to you.

**Comment by no-stack-dub-sack** [comment]
@raisedadead Where, in the entire backend section? Or just in the section that I've already modified.  Because as you know, I've only done basic node and express here, and the entire backend curriculum has code examples that also need to be wrapped in blockquotes. So I can do it for this section, but I think maybe the other sections should have everything done at once

**Comment by no-stack-dub-sack** [comment]
@raisedadead Ok, I've made some updates, I think I caught most of everything, but not sure if some things like POST and GET should be wrapped in code tags to have them stand apart even when they're not explicitly in the context of code. They are also mentioned in this section several times before they're actually introduced, so maybe this could use some re-working and DFN tags in the right places. 

This also will now close #13168 which I opened a while ago.

**Comment by no-stack-dub-sack** [comment]
@raisedadead Did you get a chance to check out the changes I made since we last checked in?

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
@no-stack-dub-sack sorry have been travelling a bit last weekend. LGTM. Lets add more formatting in further PR's.

I am yet to pull down locally and merge it in. I am okay if anyone beats me to it.

**Pull request merged**
